docs/extensibility/form-section.md

```diff
@@ -390,7 +390,7 @@ KeyValuePair widgets render an `object` value as a list of fields. One is used f
   - **type** - a string that specifies the type of the value input. The options are `object`, `number`, `text`. Defaults to `text`.
   - **keyEnum[]** - an array of options to generate a key input field with a dropdown only if the `type` is set to `object`.
   - **valueEnum[]** - an array of options to generate a value input field with a dropdown.
-- **inputInfo** - a string below the input field that shows how to fill in the input.
+- **inputInfo** - a string below the input field that shows how to fill in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.
```

Comment:
```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.
```